db/submission_test.go

```diff
@@ -59,3 +62,72 @@ func (this *DBTests) DBTestRemoveSubmission(test *testing.T) {
         }
     }
 }
+
+func (this *DBTests) DBTestFetchAttempts(test *testing.T) {
+    ResetForTesting();
+    defer ResetForTesting();
+
+    assignment := MustGetTestAssignment();
+
+    // Fetch attempts from another user who has submissions.
+    studentAttempts, err := GetSubmissionAttempts(assignment, "student@test.com");
+    if err != nil {
+        test.Fatalf("Failed to get student attempts when should be a success: %v", err);
```

Comment:
s/%s/'%v'./

Usually we single quote errors and end with a period.
Note that this is just our convention, not a go pattern.
And we are not very attached to it, we may change in the future.

This happens in a few places in this file.